packages/client/__tests__/AuthenticationPasswordReset.test.tsx

```diff
@@ -967,6 +1046,134 @@ cases<{
     },
 );
 
+cases<{
+    target: {
+        'Rendering password reset code screen': {
+            'Requested password reset code resending':
+                | 'Displaying reached rate limit toast'
+                | 'Displaying success toast'
+                | 'Displaying unknown error toast';
+        };
+    };
+    events: EventFrom<typeof passwordResetMachine>[];
+}>(
+    'Request password reset email resending',
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement -->
**🤓 nitpick:** Should we update test group label ?

Reply:
How should we rename the group? These tests are about resending an email for password reset.

Reply:
Indeed ! My bad nevermind